lidarts/statistics/routes.py

```diff
@@ -161,7 +159,7 @@ def x01():
         # sum up average per game stats
         game_average = round((current_game_stats['total_score'] / (current_game_stats['darts_thrown'])) * 3, 2) \
             if current_game_stats['darts_thrown'] else 0
-        stats['averagepergame'].append(game_average)
+        stats['averagepergame'].insert(0, game_average)
```

Comment:
As far as I know insert is a rather expensive operation on lists. It's probably better to append all elements and invert the list after the loop if that's an option.

Reply:
Yes, i will change it